.github/workflows/release.yml

```
@@ -0,0 +1,68 @@
+name: Publish SDK to NPM
+
+on:
+  release:
+    types: [published, edited]
+
+jobs:
+  publish:
+    name: Publish to NPM
+    runs-on: ubuntu-latest
+    if: ${{ ! github.event.release.draft }}
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v4
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
```

```suggestion
          node-version: 18
```

16 is already EOL and 18 is in maintenance...but 20 is another Jira ticket.

---

Yeah, but according to product, we support one previous version to the current active version, which means we support 16. So I try to do everything with that version

---

Oh I took that to mean that we support the current SDK version and the 1 SDK version prior.

---

Resolving. We can stick with 16 until we get moving towards 20.